charts/quickwit/values.yaml

```diff
@@ -147,6 +149,8 @@ indexer:
 
   serviceAnnotations: {}
 
+  serviceType: {}
```

Comment:
Same comment, we can have something like:

```yaml
indexer:
   service:
     # type: ClusterIP
     annotations: {}
```

Instead of `serviceAnnotations` and `serviceType`